src/components/LoginPanel.jsx

```diff
@@ -49,10 +49,8 @@ class LoginPanel extends Component {
 
   handleSignout = () => {
     this.props.currentUser.globalSignOut({
-      onSuccess: (result) => {
-        console.log('handleSignout result: ' + result)
+      onSuccess: () => {
```

Comment:
oh cool, i didn't realize the callback method could omit the param without causing issues for [a caller that supplies one](https://github.com/aws-amplify/amplify-js/blob/master/packages/amazon-cognito-identity-js/src/CognitoUser.js#L1634-L1656), but JS is even more tolerant of parameter mismatches than i remembered.

Reply:
Yes, very much so!